docs/GettingStarted.md

```diff
@@ -3,47 +3,252 @@
+- [Create a resource group](https://learn.microsoft.com/cli/azure/manage-azure-groups-azure-cli#create-a-resource-group) for the deployment in a region
 
-- Deploy and configure the Azure Functions
-  - [Deploy and configure Telemetry Platform](../src/TelemetryPlatform/Functions/README.md) deploys and configures the message processing code
-  - [Deploy and configure Fleet Integration](../src/FleetIntegration/Functions/README.md) deploys and configures the dataverse integration code
+``` bash
+az group create --name ${RG_FLEETINTEGRATION} --location ${LOCATION}
+```
+
+- Get the name of the event hub created in the telemetry platform.
+
+``` bash
+export tpeventhubname=$(az eventhubs namespace list --resource-group ${RG_TELEMETRYPLATFORM} --query "[].name" --output tsv)
+```
+
+- Execute the main.bicep refering to your resource group and using the event hub namespace from the telemetry platform as an argument.
+
+``` bash
+az deployment group create --resource-group ${RG_FLEETINTEGRATION} --template-file ./main.bicep --parameters evhnsTelemetryPlatformNamespaceName=${tpeventhubname}
+```
+
+You can try out sending messages and the integration with Azure Data Explorer.
+
+#### Deploy the telemetry platform functions for connection to the dataverse
+
+> [!NOTE]
+> This step requires a Dynamics 365 installation with Field Service. You can skip this step in case you don't have an installation available.
+
+Follow the instructions in [Dataverse Integration](DataverseIntegration.md) to automatically create ```IoT Event``` entities.
+
+## Try it out
+
+### Send sample messages
+
+The MQTT Test Client will send sample payloads using the generated certificates. The client will create 5 connections to the MQTT broker functionality
+of Event Grid, and send a JSON payload every second per device.
+
+The payloads can be found in the ./SamplePayloads directory. It consists of a JSON array that contains several values, harmonized using the
+Vehicle Signal Specification (VSS) from COVESA.
+
+Change your directory to ```./src/TestClient``` and build the client
+
+```bash
+dotnet build
+```
+
+To execute, set an environment variable with the MQTT host name and run using dotnet.
+
+```bash
+export gw_url=$(az eventgrid namespace list --resource-group ${RG_TELEMETRYPLATFORM} --query [].topicSpacesConfiguration.hostname --output tsv)
+dotnet run
+```
+
+If you prefer to use containers, you can build the Test Client in a container using:
+
+```bash
+docker build -t test-client-image -f Dockerfile ../..
+```
+
+> [!WARNING] 
+> The image will include the client certificates. This is not intented for production use.
+
+Then use the following command to run locally.
+
+```bash
+docker run -it -e gw_url=${gw_url} --rm test-client-image
+```
+
+### Monitor the behaviour of the azure functions
+
+You can connect to the azure functions streams from the Portal or using  the following command:
```

Comment:
```suggestion
You can connect to the Azure Functions log stream from the portal or by using the following command:
```

Reply:
Can we give any more information here about what they will see when they look at this? i.e. why would it be interesting/important to look at the log stream? Would you call their attention to anything in particular?